pkg/apis/planetscale/v2/etcdlockserver_types.go

```diff
@@ -113,6 +113,9 @@ type EtcdLockserverTemplate struct {
 	// that will be run to completion one after another before any app containers are started.
 	InitContainers []corev1.Container `json:"initContainers,omitempty"`
 
+	// SidecarContainers can optionally be used to supply extra sidecar containers
```

Comment:
```suggestion
	// SidecarContainers can optionally be used to supply extra containers that run alongside the main containers.
```

Reply:
Cool, will update everywhere and then merge.